src/components/Search.js

```diff
@@ -0,0 +1,89 @@
+        :root {
+          --docsearch-primary-color: #ff6b81;
+          --docsearch-text-color: rgb(245, 246, 247);
+          --docsearch-container-background: rgba(9, 10, 17, 0.8);
+          --docsearch-modal-background: #281e36;
+          --docsearch-modal-shadow: inset 1px 1px 0 0 rgb(44, 46, 64),
+            0 3px 8px 0 rgb(0, 3, 9);
+          --docsearch-searchbox-background: rgb(9, 10, 17);
+          --docsearch-searchbox-focus-background: #000;
+          --docsearch-hit-color: rgb(190, 195, 201);
+          --docsearch-hit-shadow: none;
+          --docsearch-hit-background: rgb(9, 10, 17);
+          --docsearch-key-gradient: linear-gradient(
+            -26.5deg,
+            rgb(114 86 111) 0%,
+            rgb(65 46 80) 100%
+          );
+          --docsearch-key-shadow: inset 0 -2px 0 0 rgb(81 31 82),
+            inset 0 0 1px 1px rgb(125 81 111), 0 2px 2px 0 rgba(3, 4, 9, 0.3);
+          --docsearch-footer-background: #2f263c;
+          --docsearch-footer-shadow: inset 0 1px 0 0 rgba(73, 76, 106, 0.5),
+            0 -4px 8px 0 rgba(0, 0, 0, 0.2);
+          --docsearch-logo-color: #fff;
+          --docsearch-muted-color: rgb(127, 132, 151);
+          --docsearch-key-shadow: inset 0 -2px 0 0 rgb(40, 45, 85),
+            inset 0 0 1px 1px rgb(81, 87, 125), 0 2px 2px 0 rgba(3, 4, 9, 0.3);
+          --docsearch-footer-background: #2f263c;
+          --docsearch-footer-shadow: inset 0 1px 0 0 rgba(73, 76, 106, 0.5),
+            0 -4px 8px 0 rgba(0, 0, 0, 0.2);
+          --docsearch-logo-color: rgb(255, 255, 255);
+          --docsearch-muted-color: rgb(127, 132, 151);
+
+          .DocSearch-Button {
+            --docsearch-searchbox-background: rgb(235, 237, 240);
+            --docsearch-searchbox-focus-background: #fff;
+            --docsearch-text-color: rgb(28, 30, 33);
+            --docsearch-muted-color: rgb(150, 159, 175);
+            --docsearch-key-gradient: linear-gradient(
+              -225deg,
+              rgb(213, 219, 228) 0%,
+              rgb(248, 248, 248) 100%
+            );
+            --docsearch-searchbox-shadow: 0 0 0 4px rgba(0, 0, 0, 0.3);
+            --docsearch-key-shadow: inset 0 -2px 0 0 rgb(205, 205, 230),
+              inset 0 0 1px 1px #fff, 0 1px 2px 1px rgba(30, 35, 90, 0.4);
+          }
+        }
+
+        .DocSearch-Button {
+          margin: 10px 10px;
+          height: 39px;
+        }
+
+        .DocSearch-Button-Placeholder {
+          width: 100%;
+          text-align: left;
+        }
+
+        .DocSearch-Modal {
+          a {
+            border-bottom-style: none;
+          }
+        }
+
+        @media (max-width: 750px) {
+          .DocSearch-Button-KeySeparator,
+          .DocSearch-Button-Key {
+            display: flex;
+          }
+
+          .DocSearch-Button-Placeholder {
+            display: flex;
+          }
+        }
+      `}
+    />
+    <DocSearch apiKey="d5fa05c4e33e776fbf2b8021cbc15b37" indexName="popper" />
```

Comment:
Even though we don't lazy load DocSearch, we should provide the integration with Gatsby, that's where DocSearch v3 shines:

- [Use the navigator API](https://github.com/francoischalifour/reactjs.org/blob/656358b1bd1b41dcd212b0db2ca285e1c3e02600/src/components/LayoutHeader/DocSearch.js#L136)
- [Use their `Link` component](https://github.com/francoischalifour/reactjs.org/blob/656358b1bd1b41dcd212b0db2ca285e1c3e02600/src/components/LayoutHeader/DocSearch.js#L141)
- [Leverage their preloading](https://github.com/francoischalifour/reactjs.org/blob/656358b1bd1b41dcd212b0db2ca285e1c3e02600/src/components/LayoutHeader/DocSearch.js#L142-L154)

Reply:
Resolved thanks to https://github.com/s-pace/website-2/pull/2/commits/3d5de3aceb96ef5bba09ef79f435183cf1ebeb36